packages/valaxy/src/node/utils/net.ts

```diff
@@ -0,0 +1,35 @@
+import net from 'net'
```

Comment:
Just refer https://github.com/slidevjs/slidev/commit/9ef11bb3bd2aaa576f131ed0b2efbdfc16c1651e.

Your code might be a bit long.